### PR TITLE
Refactor watchdog

### PR DESCRIPTION
**Pull request by DmytroKushnirSoftserveinc**
watchdog refactored using C++ best practices guidelines https://github.com/leggedrobotics/ros_best_practices/blob/master/ros_package_template

### CODE REVIEW

File: velocity_topic_watchdog/package.xml [hunk 1]
```diff
@@ -1,62 +1,20 @@
...
-  <!-- Use exec_depend for packages you need at runtime: -->
-  <!--   <exec_depend>message_runtime</exec_depend> -->
-  <!-- Use test_depend for packages you need only for testing: -->
-  <!--   <test_depend>gtest</test_depend> -->
-  <!-- Use doc_depend for packages you need only for building documentation: -->
-  <!--   <doc_depend>doxygen</doc_depend> -->
   <buildtool_depend>catkin</buildtool_depend>
   <depend>geometry_msgs</depend>
   <depend>roscpp</depend>
   <depend>rospy</depend>
```
Comment (ldemk) [inline velocity_topic_watchdog/package.xml]: Remove rospy, please

File: velocity_topic_watchdog/package.xml [hunk 2]
```diff
@@ -1,62 +1,20 @@
...
 
-
-  <!-- The export tag contains other, unspecified, tags -->
   <export>
```
Comment (ldemk) [inline velocity_topic_watchdog/package.xml]: Remove this tag

**Review by ldemk — CHANGES_REQUESTED** [review]

File: velocity_topic_watchdog/package.xml (hunk 1 above)
Comment (DmytroKushnirSoftserveinc) [reply velocity_topic_watchdog/package.xml]: commit dfa1a66bd68d103aed25e963542d79f54a66ce73

**Comment by DmytroKushnirSoftserveinc** [comment]
#24 #29  closed

**Pull request merged**